enzy_htp/analysis/rmsd.py

```diff
@@ -0,0 +1,120 @@
+#! python3
+# -*- encoding: utf-8 -*-
+'''
+@File    :   rmsd.py
+@Created :   2024/07/12 18:45
+@Author  :   Zhong, Yinjie
+@Version :   2.0
+@Contact :   yinjie.zhong@vanderbilt.edu
+'''
+
+# Here put the import lib.
+from typing import List, Dict, Union
+
+# Here put EnzyHTP imports.
+from enzy_htp import interface as eh_interface, _LOGGER
+from enzy_htp import config as eh_config
+from enzy_htp._interface.pymol_interface import PyMolInterface
+from enzy_htp._interface.amber_interface import AmberInterface
+from enzy_htp.core import file_system as fs
+from enzy_htp.structure import Residue, Structure
+from enzy_htp.structure.structure_ensemble import StructureEnsemble
+
+ACCEPTED_INTERFACES: Dict[str, Union[PyMolInterface, AmberInterface]] = {
+    "pymol": eh_interface.pymol,
+    "amber": eh_interface.amber,
+}
+
```

Comment:
The API referencing here does not make sense. Please reference how we did this in for example mutation.api.py::MUTATE_STRU_ENGINE

Reply:
Resolved.